File: shoestorefe/lib/domain/entities/supplier.dart
class Supplier {
  final int id;
  final String? code;
  final String name;
  final String? contactInfo;
  final int statusId;

  Supplier({
    required this.id,
    this.code,
    required this.name,
    this.contactInfo,
    required this.statusId,
  });
}

File: ShoeStore/ShoeStore.Domain/Entities/Supplier.cs
namespace ShoeStore.Domain.Entities;

public class Supplier
{
    public int Id { get; set; }
    public string? Code { get; set; }
    public string Name { get; set; } = null!;
    public string? ContactInfo { get; set; }
    public int StatusId { get; set; }

    public Status Status { get; set; } = null!;
    public ICollection<Product>? Products { get; set; }
}
